### PR TITLE
feat: create my-account library

### LINKED ISSUE #7960: Create my-account lib
As seen in https://github.com/SAP/spartacus/issues/7959, we will start to introduce more smaller feature-libraries to Spartacus codebase, and we will start with `my-account`.

### PR DESCRIPTION
**Pull request by znikola**
closes #7960

### CODE REVIEW

File: feature-libs/my-account/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@spartacus/myaccount",
```
Comment (marlass) [inline feature-libs/my-account/package.json]: What about `@spartacus/my-account` name? 
We need to specify naming convention for the libs

Comment (znikola) [reply feature-libs/my-account/package.json]: Agreed. Added convention section to the readme and changed to `my-account`.

File: docs/libs/creating-lib.md [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+  '@spartacus/my-account': 'feature-libs/my-account',
+};
+```
+
+- `.github/api-extractor-action/api-extractor-for-branch.sh`
+
+Add the following (replace the `my-account` and `MY_ACCOUNT_CONFIG_PATH` with the name of your lib):
+
+```sh
+MY_ACCOUNT_CONFIG_PATH="./../.github/api-extractor-action/api-extractor-configs/api-extractor.json"
```
Comment (marlass) [inline docs/libs/creating-lib.md]: This can be dropped now.

File: scripts/changelog.ts [hunk 3]
```diff
@@ -76,6 +76,7 @@ export default async function run(
     '@spartacus/schematics': 'projects/schematics',
     '@spartacus/incubator': 'projects/incubator',
     '@spartacus/cds': 'projects/cds',
+    '@spartacus/my-account': 'feature-libs/my-account',
```
Comment (marlass) [inline scripts/changelog.ts:79]: We also have to update packages.ts file.

**Review by marlass — CHANGES_REQUESTED** [review]
We should also add simple README to the lib, so it won't have blank page on npm

Another step would be setting up repository for nightly versions (requires help from Gil and can be done a later)

File: scripts/changelog.ts (hunk 3 above)
Comment (marlass) [reply scripts/changelog.ts:79]: We have also one more place in this file with libs decalration at the end

**Review by marlass — CHANGES_REQUESTED** [review]

**Review by marlass — CHANGES_REQUESTED** [review]
I looked at other projects files and found 2 more things:
- sonarcloud configuration needs adjustment
- prettier scripts in package.json have defined paths to `projects`

File: sonar-project.properties [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
+sonar.sources=projects/assets,projects/cds,projects/core,projects/storefrontstyles,projects/storefrontlib,projects/schematics,projects/incubator,projects/vendor,feature-libs/my-account
 sonar.exclusions=**/node_modules/**
 
-sonar.tests=projects/cds,projects/core,projects/storefrontlib
+sonar.tests=projects/cds,projects/core,projects/storefrontlib,feature-libs/my-account
 sonar.test.inclusions=**/*.spec.ts
 
-sonar.typescript.lcov.reportPaths=coverage/cds/lcov.info,coverage/core/lcov.info,coverage/storefront/lcov.info
-sonar.javascript.lcov.reportPaths=coverage/cds/lcov.info,coverage/core/lcov.info,coverage/storefront/lcov.info
+sonar.typescript.lcov.reportPaths=coverage/cds/lcov.info,coverage/core/lcov.info,coverage/storefront/lcov.info,feature-libs/my-account/lcov.info
```
Comment (marlass) [inline sonar-project.properties]: I think this doesn't match the path you have in karma config

**Review by marlass — CHANGES_REQUESTED** [review]

File: sonar-project.properties (hunk 4 above)
Comment (znikola) [reply sonar-project.properties]: Thanks, I set it to `coverage/my-account/lcov.info`

**Review by marlass — APPROVED** [review]

**Pull request merged**